interfaces/HO-Portal/cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+    "baseUrl": "http://localhost:4200"
```

Comment:
I think you're local set-up hasn't used/configured to use Prettier on these files, as the whitespace differs here and there...

If you open the folder `interfaces/HO-Portal` as a 'workspace' in VSCode (with the Prettier-plugin installed), it will automatically pick up the configuration to "format on save" for all supported files.

Or manually running `npm run fix:syntax` before committing.